src/OpenTelemetry/Sdk.cs

```diff
@@ -83,5 +84,19 @@ public static TracerProviderBuilder CreateTracerProviderBuilder()
         {
             return new TracerProviderBuilderBase();
         }
+
+        /// <summary>
+        /// Creates a <see cref="LoggerProviderBuilder"/> which is used to build
+        /// an <see cref="LoggerProvider"/>. In a typical application, a single
+        /// <see cref="LoggerProvider"/> is created at application startup and
+        /// disposed at application shutdown. It is important to ensure that the
+        /// provider is not disposed too early.
+        /// </summary>
+        /// <returns><see cref="LoggerProviderBuilder"/> instance, which is used
+        /// to build a <see cref="LoggerProvider"/>.</returns>
+        internal static LoggerProviderBuilder CreateLoggerProviderBuilder()
+        {
+            return new LoggerProviderServiceCollectionBuilder();
```

Comment:
Could we rename `LoggerProviderServiceCollectionBuilder` to `LoggerProviderBuilderBase` to stay consistent with Traces and Metrics?